scikit-learn-tqdm-roadmap.md

```diff
@@ -0,0 +1,29 @@
+# scikit-learn tqdm Integration Roadmap
+### Background
+tqdm is a progress bar in Python.  See more information on the tqdm website ➡ https://tqdm.github.io/.  Currently tqdm has integration with pandas, concurrent.futures, itertools.product, keras, and telegram.  **This project aims to add scikit-learn to the previous list by integrating tqdm with scikit-learn as much as possible.**
+<!-- What the hell is telegram lol? -->
```

Comment:
https://github.com/tqdm/tqdm/pull/949 :)

Reply:
Oof haha, that's super cool :)
Thanks for enlightening me